### PR TITLE
Various small improvements

### PR DESCRIPTION
**Pull request by pizkaz**
* Updated / improved docker-compose.prod.yaml + .env.prod.template
* Do not show the following warning message to admins:
  2024-09-30-170704_989x361_scrot
* Fixes i18n in the ObjectDetails modal:
  2024-09-30-175228_705x375_scrot

### CODE REVIEW

File: contrib/docker/.env.prod.template [hunk 1]
```diff
@@ -247,14 +248,13 @@ SECURITY_EMAIL=
 
 # MAP EDITOR SETTINGS
 ENABLE_MAP_EDITOR=true
-# Enable broadcast areas in map editor (beta)
+# Enable broadcast areas in map editor
```
Comment (moufmouf) [inline contrib/docker/.env.prod.template:251]: This is still pretty much beta.

We have had a number of issues and are not 100% sure this is stable and scales well. For instance, it's not yet enable in our cloud offering.

```suggestion
# Enable broadcast areas in map editor (beta)
```

**Review by moufmouf — COMMENTED** [review]
Thanks a lot for the PR!

Just one little comment, I believe the "beta" flag for the broadcast areas in the map editor is still needed. Did you successfully use those on a large scale?

File: contrib/docker/.env.prod.template (hunk 1 above)
Comment (pizkaz) [reply contrib/docker/.env.prod.template:251]: Ah, that explains a lot. I just assumed, it must be well tested by now and attributed issues with it to my inability to setup stuff correctly. ^^

**Comment by moufmouf** [comment]
Thanks!

**Pull request merged**